### ISSUE TITLE
Yarn version manager (yvm)

### ISSUE THREAD

**Issue by jakebolam** [yarnpkg/website#848]
Thoughts on adding links to using a yarn version manager? like https://yvm.js.org/

**Comment by Daniel15** [comment]
Seems like an interesting idea. The installation script for yvm doesn't do any integrity checking (no SHA hash, no GPG signature) which is a bit concerning. I wonder if something like this should actually be built in to Yarn itself - I think there's already *some* support for using a different Yarn version per project, but it involves committing the yarn.js file to source control.

**Comment by jakebolam** [comment]
Great point. Will start looking at the security thing (we're currently working on the version thing too). 

Yeah the idea was that by keeping it separate, yarn can do more forward, drastic changes, between versions (that may even output different lockfiles). Currently managing yarn across our 100+ repos (if we have to upgrade them all at once) is massively interruptful.

**Comment by keego** [comment]
Any update on this since tophat/yvm#102 was merged and shipped in tophat/yvm#131?

Wondering if its safe enough to move to yvm (from a security standpoint) or if I should still recommend my team use the [scan-homebrew-prs-for-the-version-you-desire approach](https://stackoverflow.com/a/54130907/3234235).

**Comment by jakebolam** [comment]
@keego the signature ensures that a mirror hasn't tampered with the yarn version

yvm installs have been steadyily increasing over the last 2 months (with around 85k installs a day now):
<img width="587" alt="screen shot 2019-02-12 at 1 05 32 pm" src="https://user-images.githubusercontent.com/3534236/52658310-b82ffb80-2ec8-11e9-8ea2-522c6c0a49cb.png">

**Comment by arcanis** [comment]
I'm going to close this one as a dupe of #896 (it would be the opposite really, but most of the discussion is there so we might as well use the one with the most comments).

**tldr:** Even though I'm sure yvm is a great project, I would personally suggest using [`yarn policies set-version`](https://yarnpkg.com/en/docs/cli/policies#toc-policies-set-version) in order to version Yarn within a project. It's native, so it's one less dependency.
